### PR TITLE
Added DAOs to the Online Communties section

### PR DESCRIPTION
**Pull request by wschwab**
## Description

DAOS got added to the Online Communities section of the Community page

DAOs are a very Ethereum-esque way of discovering and interacting with the Ethereum community. In order to help people find them, a list of DAOs was added to the Online Communities section, using websites and Twitter handles.

### CODE REVIEW

**Comment by jjmstark** [comment]
Made a few changes to this PR:

- Changed the location of the DAO information - generally we use a "first in first out" principle, and it also doesn't really make senes to group these with forums or chat rooms, since DAOs are much different then those. Gave them their own category further down the page.
- Re-arranged into alphabetical order
- I removed D5, as it does not really look like a DAO, but rather a consulting company (https://d5.ai/)

@wschwab thoughts?

**Comment by wschwab** [comment]
Thanks for the quick review @jjmstark  !

I would argue for including D5. They are a consultancy by all appearances, but one governed by a DAO. (Scroll down, they're on Aragon.) I get your distinction between businesses and forums, but I think the line is a bit gray with DAOS, and especially if they're getting their own category,

Other than that, sounds great!

File: docs/community/index.md [hunk 1]
```diff
@@ -180,6 +181,20 @@ Interested in starting your own meetup? Check out the [BUIDL Network](https://co
...
+
+  - [LexDAO](https://lexdao.org) [@lex_DAO](https://twitter.com/lex_DAO) -  _Legal engineering_
+  - [Machi X](https://machix.com) [@MachiXOfficial](https://twitter.com/MachiXOfficial) - _Art community_
+  - [MarketingDAO](https://marketingdao.org/) [@MarketingDAO](https://twitter.com/MarketingDAO) - _Community focused on marketing Ethereum_
+  - [MetaCartel](https://metacartel.org) [@Meta_Cartel](https://twitter.com/Meta_Cartel) - _DAO incubator_
+  - [MetaCartel Ventures](https://metacartel.xyz) [@VENTURE_DAO](https://twitter.com/VENTURE_DAO) - _Venture for pre-seed crypto projects_
+  - [MetaClan](https://discord.gg/euUeZVp) [@MetaClanDAO](https://twitter.com/MetaClanDAO) - _esports_
+  - [MolochDAO](https://molochdao.com) [@MolochDAO](https://twitter.com/MolochDAO) - _Community focused on funding Ethereum development_
+  - [ΜΓΔ](https://daohaus.club/dao/v1/0x1b3d7efb93ec432b0d1d56880c23303979b379e9) (Meta Gamma Delta) [@metagammadelta](https://twitter.com/metagammadelta) - _Women-led projects_
+  - [Raid Guild](https://raidguild.org) [@RaidGuild](https://twitter.com/RaidGuild) - _Web3 devs_
```
Comment (hammadj) [inline docs/community/index.md:196]:
```suggestion
  - [Raid Guild](https://raidguild.org) [@RaidGuild](https://twitter.com/RaidGuild) - _Web3 devs_
  - [MetaGame](https://metagame.wtf) [@MetaFam](https://twitter.com/MetaFam) - _MMORPG Game Mechanics for Real Life_
  - [MetaFactory](https://metafactory.ai) [@TheMetaFactory](https://twitter.com/TheMetaFactory) - _Digiphysical Apparel Brands_
```

Comment (wschwab) [reply docs/community/index.md:198]: @hammadj I hope you don't mind, but I recommitted your suggestion keeping in alphabetical order. I was trying to figure if there was a way that I could keep it as your commit, but if there is, I wasn't able to figure out how.

**Comment by jjmstark** [comment]
> Thanks for the quick review @jjmstark !
> 
> I would argue for including D5. They are a consultancy by all appearances, but one governed by a DAO. (Scroll down, they're on Aragon.) I get your distinction between businesses and forums, but I think the line is a bit gray with DAOS, and especially if they're getting their own category,
> 
> Other than that, sounds great!

Yup, I agree that the definition of a DAO can be a bit hazy. So instead, we should think about the purpose of this page. Ultimately the goal of ethereum.org/community is to be a launchpad for people to join the Ethereum community. To do this, we list various entry points (like events, meetups, and forums) or specific sub-communities (like r/ethfinance). 

The idea being considered here is _not_ whether we should list everything that is technically a DAO on this page. Rather, we're considering how to include a list of DAO communities that are themselves good entry points or Ethereum sub-communities. 

My objection to listing D5 isn't about whether it is technically a DAO, or anything about the company or the people involved. Rather, I want to ensure that listing it would serve the purpose of this page. Can anyone join D5? Is D5 a growing community of people interested in Ethereum? Where can someone go to participate in the D5 community? These are some of the questions we should consider when deciding whether it make sense to list a DAO community on this page.

**Comment by wschwab** [comment]
@jjmstark  If the only issue right now is D5, which you already took out in your commit, can we merge this, and then add D5 later if we see that it meets the criteria? 

(I'd like to do some research into D5, but this PR has been hanging for a week already, and I'd certainly rather see it get merged, and make a new PR to add on to it later.)

**Comment by samajammin** [comment]
@all-contributors please add @wschwab for content

**Comment by samajammin** [comment]
@all-contributors please add @hammadj for content

**Pull request merged**